### ISSUE TITLE
Fix tests disabled during package:analyzer migration

### ISSUE THREAD

**Issue by scheglov** [dart-lang/sdk#44522]
This should be tests that are written in a more unit like fashion, and don't always set all expected nodes, elements, and types.

**Comment by srawlins** [comment]
Hi @scheglov , do you know if the relevent tests are fixed? You don't mention which they are.

**Comment by scheglov** [comment]
Yeah, I guess there were many of them while I was migrating the analyzer.
Search for `44522` in the analyzer.
We still have a few.

**Comment by srawlins** [comment]
Only two left!
